src/main/webapp/i18n/en/overview.json

```
@@ -4,7 +4,8 @@
             "teams": {
                 "title": "Teams",
                 "preExpiresIn": "expires in",
-                "postExpiresIn": "days"
+                "postExpiresIn": "days",
+                "official": "official"
```

According to the ticket this should be:
```suggestion
                "official": "Project team"
```